cpu/cortexm_common/ldscripts/cortexm_base.ld

```diff
@@ -108,6 +108,8 @@ SECTIONS
     /* stack section */
     .stack (NOLOAD):
     {
+        KEEP (*(.puf_stack))
+        . = ALIGN(8);
```

Comment:
No need to align twice

Reply:
Addressed. Damn, when you do things in a hurry ...